IBPSA/Fluid/Interfaces/TwoPortHeatMassExchanger.mo

```diff
@@ -17,10 +17,7 @@ model TwoPortHeatMassExchanger
   // Dynamics
   parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.DynamicFreeInitial
     "Type of energy balance: dynamic (3 initialization options) or steady state"
-    annotation(Evaluate=true, Dialog(tab = "Dynamics", group="Equations"));
-  parameter Modelica.Fluid.Types.Dynamics massDynamics=energyDynamics
```

Comment:
I don't see the need for removing massDynamics but in case I ever see a need for a separate parameter in this model then we can still add it back :)

Reply:
Yes, I also feel better add it back if there is a use case. Removing it makes it simpler for the non-expert user to understand.